CPlusPlus/Sample_Source/Text/TextSelectEnum/TextSelectEnum.cpp

```diff
@@ -0,0 +1,188 @@
+//
+// Copyright (c) 2018, Datalogics, Inc. All rights reserved.
+//
+// For complete copyright information, refer to:
+// http://dev.datalogics.com/adobe-pdf-library/license-for-downloaded-pdf-samples/
+//
+// The TextSelectEnum sample demonstrates how the Library is able to select all of the 
+// text present in an area of the page. 
+//
+// Command-line:  <output-file>     (Optional)
+//
+// For more detail see the description of the UnicodeText sample program on our Developer�s site, 
+// http://dev.datalogics.com/adobe-pdf-library/sample-program-descriptions/c1samples#TextSelectEnum
+
+#include "APDFLDoc.h"
+#include "InitializeLibrary.h"
+#include "DLExtrasCalls.h"
+#include "ASExtraCalls.h"
+#include "PagePDECntCalls.h"
+#include "PEWCalls.h"
+#include "PERCalls.h"
+#include <vector>
+
+#define DIR_LOC "../../../../Resources/Sample_Input/"
+#define DEF_INPUT "TextSearch.pdf"
```

Comment:
Is this sample supposed to be able to be run straight out of the box?  This Resources folder does not exist.